src/monodroid/jni/monodroid-glue.cc

```diff
@@ -2422,7 +2432,7 @@ JNI_OnLoad (JavaVM *vm, void *reserved)
 JNIEXPORT void JNICALL
 Java_mono_android_Runtime_init (JNIEnv *env, jclass klass, jstring lang, jobjectArray runtimeApksJava,
                                 jstring runtimeNativeLibDir, jobjectArray appDirs, jobject loader,
-                                [[maybe_unused]] jobjectArray externalStorageDirs, jobjectArray assembliesJava, [[maybe_unused]] jstring packageName,
+                                [[maybe_unused]] jobjectArray externalStorageDirs, jobjectArray assembliesJava, jbyteArray mappingXml, jint mappingXmlLen, [[maybe_unused]] jstring packageName,
```

Comment:
If you add new parameters here, does that break the Android designer? I think that is why we made `Java_mono_android_Runtime_initInternal `.

Reply:
Fixed.  Extra parameters removed from `Runtime.initInternal()`, designer tests now passing.